apps/wing/src/wingc.ts

```diff
@@ -25,12 +25,14 @@ export interface WingCompilerLoadOptions {
 
 export async function load(options: WingCompilerLoadOptions) {
   const preopens = {
+    "/": "/",
     // .jsii access
     [WINGSDK_MANIFEST_ROOT]: WINGSDK_MANIFEST_ROOT,
     ...(options.preopens ?? {}),
   } as Record<string, string>;
 
   if (process.platform === "win32") {
+    preopens["C:\\"] = "C:\\";
```

Comment:
tbh it may be impossible to grant "full access" to windows, until we delegate fs access to node we will have issues here

Reply:
Do we have an issue about this?

Reply:
https://github.com/winglang/wing/issues/1560